pilot/pkg/serviceregistry/kube/controller/ambient/waypoints.go

```diff
@@ -146,7 +146,7 @@ func fetchWaypointForWorkload(ctx krt.HandlerContext, Waypoints krt.Collection[W
 // defaultNamespace avoids the need to infer when object meta from a namespace was given
 func getUseWaypoint(meta metav1.ObjectMeta, defaultNamespace string) (named *krt.Named, isNone bool) {
 	if labelValue, ok := meta.Labels[constants.AmbientUseWaypointLabel]; ok {
-		if labelValue == "#none" || labelValue == "~" {
+		if labelValue == "none" {
```

Comment:
```suggestion
                // NOTE yes, this means waypoints named `none` will be ignored.
                // Don't name your waypoints that.
		if labelValue == "none" {
```

Reply:
seemed a bit snarky to me but I did try a revised wording with, I think, similarly helpful information